### ISSUE TITLE
user setting will force overwrite after install

### ISSUE THREAD

**Issue by foxundermoon** [GabrielBB/vscode-lombok#20]
before installing the extension, there is many customs configuration.

but now all old configuration lost, and it is the new configuration.

```
{
  "java.jdt.ls.vmargs": "-noverify -Xmx1G -XX:+UseG1GC -XX:+UseStringDeduplication -javaagent:\"/Users/fox/.vscode/extensions/gabrielbb.vscode-lombok-0.9.7/server/lombok.jar\" -Xbootclasspath/a:\"/Users/fox/.vscode/extensions/gabrielbb.vscode-lombok-0.9.7/server/lombok.jar\""
}
```

**Comment by foxundermoon** [comment]
do not edit user setting file by the programming.

**Comment by GabrielBB** [comment]
Thanks for reporting it. I will check it out

**Comment by jrussellsmyth** [comment]
this also causes problems for users using VSCode Remote Development support (https://code.visualstudio.com/docs/remote/remote-overview) - if vscode-lombok is installed outside of a devcontainer or inside of one, any containers without lombok support installed will fail to start java language server due to the above config.

**Comment by GabrielBB** [comment]
@jrussellsmyth I don’t have experience using Remote Development in VSCode. Do you know why it fails? I assume it is because the container can’t resolve the jar path that is being appended to the config, right?

**Comment by jrussellsmyth** [comment]
@GabrielBB that is correct, you are adding that config at a global level, and any config that does not include your plugin does not have that jar in the location you expect it. 

The reason or using remote development for most is to use isolated Docker containers as development environment to isolate each projects configuration. but because your plugin puts this config in the global space, it is active even when the filesystem is different.

This even happens to the main non-remote project if you add the plugin in a remote project, again, as the config change is at the global level.

**Comment by jeyjeyemem** [comment]
@GabrielBB I'm also encountering the same issue with remote container not being able to find the lombok jar inside the dev container. I don't even use lombok on this specific project but the extension is causing the java server to crash.

**Comment by 0xVesion** [comment]
I'm also running into this issue when using devcontainers. The correct path inside a dev container is: `/home/vscode/.vscode-server/extensions/gabrielbb.vscode-lombok-1.0.1/server/lombok.jar`
